nncf/quantization/algorithms/accuracy_control/backend.py

```diff
@@ -51,6 +51,16 @@ def get_quantizable_metatypes() -> List[OperatorMetatype]:
         :return: The list of metatypes for operations that may be quantized.
         """
 
+    @staticmethod
+    @abstractmethod
+    def get_graph_inputs(nncf_graph: NNCFGraph) -> List[NNCFNode]:
```

Comment:
Please tell me explicitly what is wrong with this method

Reply:
This method should not have been introduced in the first place, see ideas in https://github.com/openvinotoolkit/nncf/pull/2276#pullrequestreview-1737353180 for how you could achieve what you want without having an extra backend-specific method like this.

Reply:
You could add a `get_components` method on an `NNCFGraph`, that would return a list of `NNCFGraphComponent`s. An `NNCFGraphComponent` would be a struct containing a corresponding connected `.component` and a list of nodes `potential_component_input_nodes` that would contain the nodes you want.

Reply:
Ok, I see that you have introduced a `get_disconnected_nodes` on an `NNCFGraph`, which is good because we now have a distinct name at least. For the method that is currently called `get_input_nodes` in the algo backend, I suggest a name `get_start_nodes_for_activation_path_tracing` - that would be descriptive and dissuade anyone from using this method to do anything else except activation/constant path filtering. Note, however, that in this way you are bringing the implementation details of constant node filtering into an `AccuracyControlAlgoBackend`, mixing the responsibilities. Consider refactoring this now or in the future, for example, as I suggested, by encapsulating constant path filtering in the `NNCFGraph`.